react/src/page_template/statistic-tree.ts

```diff
@@ -63,8 +64,9 @@ export const statsTree: StatsTree = rawStatsTree.map(category => (
             ...group,
             contents: group.contents.map(({ year, stats_by_source }) => ({
                 year,
-                stats: stats_by_source.map(({ name, stats: s }) => ({
+                stats: stats_by_source.map(({ name, indentedName, stats: s }) => ({
                     name,
+                    indentedName: indentedName ?? undefined,
```

Comment:
```suggestion
                    indentedName,
```

surprised there's not a lint rule for this

Reply:
It's null in the model!

Reply:
Oops the Github collapse got me